scripts/v2/make-mi-fic.sh

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env bash
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+function create_role_assignment() {
+  az role assignment create --assignee "${USER_ASSIGNED_OBJECT_ID}" \
+      --role "Owner" \
+      --subscription "${AZURE_SUBSCRIPTION_ID}"
+}
+
+function retry_create_role_assignment() {
+  until create_role_assignment; do
+    sleep 5
+  done
+
+}
+
+print_usage() {
+  echo "Usage: make-mi-fic-kind-wi-storage.sh -g <RESOURCE_GROUP> -i <ISSUER> -d <DIR>"
+}
+
+while getopts 'g:i:d:' flag; do
+  case "${flag}" in
+    g) RESOURCE_GROUP="${OPTARG}" ;;
+    i) ISSUER="${OPTARG}" ;;
+    d) DIR="${OPTARG}" ;;
```

Comment:
Don't you need to predefine these variables (above the loop) in order to prevent inheriting random(ish) env vars?

Reply:
Probably yes. Will fix